### DISCUSSION TITLE
How to PEST Filament's Authentication Proccess and assert  that authenticated user sees the admin page?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#5929] [Help] [answered]
How to PEST Filament's Authentication Proccess and assert  that authenticated user sees the admin page?

I have Jetstream installed.

**Comment by danharrin** [comment, marked as answer]
We use Laravels authentication system so you should just be able to use `actingAs()` and ensure that your User implements FilamentUser.

**Reply by otavio-araujo** [reply]
Thank you Dan!
